e2e_samples/parking_sensors/scripts/deploy_azdo_pipelines.sh

```diff
@@ -44,6 +44,30 @@ github_sc_name="${PROJECT}-github"
 github_sc_id=$(az devops service-endpoint list --output json |
     jq -r --arg NAME "$github_sc_name" '.[] | select(.name==$NAME) | .id')
 
+ifexistsoverwrite() {
```

Comment:
should this function be at the top of the script? or maybe in the `common.sh` script?

Reply:
the function is following the same structure as it was before, right? So the create pipeline function it was there already ( did not create). I just add a second function which is the overwrite if exist. So I am following the same structure. as before we can change, if it is case. is it the case? will we change for all the sh scripts?
@ydaponte should we move all the functions in the scripts to the common.sh.? in that case we need to refactor the other scripts too.  So we follow the same pattern

Reply:
agreed, my whole point of this comment is code cleanliness

Reply:
@LiliamLeme and @elenaterenzi  - I like the idea of moving to the common.sh script - however, I would add an issue to the backlog to address the others. For now I think you can move this one as part of the scope you are working on and we can later move the others. Sounds good?

Reply:
ok. So I will move both Create Pipeline(old)  and the new one I added.
And the others from the sh scripts will be addressed in the backlog., as you mentioned.. @ydaponte  maybe worth to mention this in our daily meetings, as it changes the process for the development. So now we have a kind of framework for bash functions.

Reply:
updated... testing now